sdks/ios/appbuilder/appbuilder.cs

```diff
@@ -230,7 +230,7 @@ public class AppBuilder
 		ninja.WriteLine ("  command = clang -isysroot $sysroot -miphoneos-version-min=10.1 -arch arm64 -c -o $out $in");
 		ninja.WriteLine ("rule gen-exe");
 		ninja.WriteLine ("  command = mkdir $appdir");
```

Comment:
Looks fine to me provided that the `GSS` framework is actually available on all platforms that this tool uses (which I'm not sure about).

Reply:
iOS 64bit devices is the only target currently